observability/README.md

```diff
@@ -44,13 +46,12 @@ sleep: 5
 -->
 
 ```bash
-cd ../hello-world
-dapr run --app-id hello-tracing --app-port 3000 node app.js
+cd ../hello-world && dapr run --app-id hello-tracing --app-port 3000 node app.js && cd ..
```

Comment:
```suggestion
cd ../hello-world && dapr run --app-id hello-tracing --app-port 3000 node app.js && cd ../observability
```

Reply:
Yeah. That makes sense. That is why I was using cd - before.